### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by codingknite**

### CODE REVIEW

**Comment by codingknite** [comment]
@t8 x @maximousblk please don't hold back any feedback.
Thanks :)

**Comment by t8** [comment]
Hey @codingknite, would you mind fixing this build issue in Vercel before we move forward here? 
image

**Comment by maximousblk** [comment]
fixed

**Comment by maximousblk** [comment]
@codingknite most of the images are invisible 🤔

**Comment by codingknite** [comment]
> @codingknite most of the images are invisible 🤔

@maximousblk this is very weird behavior that seems to be with Chrome.
Images render fine in firefox. Chrome shows they exist on the page, they just don't render 🤔

**Comment by maximousblk** [comment]
#### 📝 Changed routes:


| [**/**](https://arconnect-web-new-g7o0pnap0-th8ta.vercel.app/) | [**/onboarding/choose-action**](https://arconnect-web-new-g7o0pnap0-th8ta.vercel.app/onboarding/choose-action) |
|:-:|:-:|
|<a href="https://arconnect-web-new-g7o0pnap0-th8ta.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/m82CvN7FfxhJgOn7oO7T591F8lzqYYsJX5qUh1tc5Z16zrSIA1J-DkSbqjP2NbX3NRr6bVKNAJ66qNGyAFEgwg.png">(view full size)</a> |<a href="https://arconnect-web-new-g7o0pnap0-th8ta.vercel.app/onboarding/choose-action">Screenshot of /onboarding/choose-action</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/GsusUYtY0FFXrtqxm7z2RfAlJdtDLs_zXmCrszPL0P0bIa-HudF0e-V6U_NeYgpkybqGtV4XCw-Pj1RDWmZcqQ.png">(view full size)</a> |



| [**/onboarding/explore**](https://arconnect-web-new-g7o0pnap0-th8ta.vercel.app/onboarding/explore) | [**/onboarding/guide**](https://arconnect-web-new-g7o0pnap0-th8ta.vercel.app/onboarding/guide) |
|:-:|:-:|
|<a href="https://arconnect-web-new-g7o0pnap0-th8ta.vercel.app/onboarding/explore">Screenshot of /onboarding/explore</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/Nf3_SrAYUdkVFYjSZCs4jQ5SZ8TaiGOtM04DaDw_i_jSRy1xVGCO5QXn9ugYOdszxgfHKfPXRd2qiZbvNv5BLg.png">(view full size)</a> |<a href="https://arconnect-web-new-g7o0pnap0-th8ta.vercel.app/onboarding/guide">Screenshot of /onboarding/guide</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/VgZo2n2yutxr7H-JsTW5n80V4T_CiGFYvE9BwmM7J7A99sSA3o7uN4zK5KKuDXL3_3mwh_qor0flgGCJ1EmWYA.png">(view full size)</a> |



| [**/onboarding**](https://arconnect-web-new-g7o0pnap0-th8ta.vercel.app/onboarding) |
|:-:|
|<a href="https://arconnect-web-new-g7o0pnap0-th8ta.vercel.app/onboarding">Screenshot of /onboarding</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/wgsCBs12X56UdQxYIf4Ne4kXG-Rf46osz_pdcS27CVJ-t1Q4OyVPW3SO-ZR_KbsGEp9RDoatZrgllSOPpRepcg.png">(view full size)</a> |







Commit 5ed450c0a1ce5611c86c643f245b1c5e19437e84 (https://arconnect-web-new-g7o0pnap0-th8ta.vercel.app).

**Comment by codingknite** [comment]
## Todo

This commit was supposed to fix all the images but some still remain stubborn. 

- [ ] Fix images
- [x] Add micro interactions to buttons

✔️ ready to merge

**Comment by t8** [comment]
@codingknite have added some comments on a new Jam.dev board. Let me know if you can't see this :)

https://jam.dev/tates-team/browser/list

**Comment by codingknite** [comment]
Thanks @t8 saw all the comments. We had an issue with the svgs in the figma file which led to the weird behavior with all the images. Max helped figure this out so we've made those changes. 
Thanks again for the feedback here 👍

**Comment by codingknite** [comment]
Not sure why the images are shrinking in production.

### development
shot2

### production
shot


@t8 x @maximousblk do you think this could have anything to do with CSS Grid

**Comment by codingknite** [comment]
@t8 just made some major fixes here. Review is welcome

File: components/Home/Features/index.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            <h3>
+              Develop <span>on ArConnect</span>
+            </h3>
+            <p className={styles.text}>
+              Built by Developers for Developers, ArConnect allows you use
+              developer friendly tools to integrate and build on ArConnect
+              across blockchains seamlessly. Explore our Developer documentation
+              to get started.
+            </p>
+            <Link href="/security">
```
Comment (t8) [inline components/Home/Features/index.tsx]:
```suggestion
            <Link href="https://docs.th8ta.org/arconnect/functions">
```

File: components/Home/Features/index.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+          handIcon1="/assets/handLeft.svg"
+          handIcon2="/assets/handRight.svg"
+          wallet
+        />
+
+        <FeatureCard
+          title="Gateway"
+          heading="Access Dapps"
+          grayedText="on the permaweb"
+          text="ArConnect is your go to forum to access Dapps and learn about the permaweb ecosystem."
```
Comment (t8) [inline components/Home/Features/index.tsx]:
```suggestion
          text="ArConnect is your go-to solution to access Dapps while learning about the permaweb ecosystem."
```

File: components/Home/Features/index.tsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+const Features = () => {
+  return (
+    <>
+      <section className={styles.featureCards}>
+        <FeatureCard
+          title="Wallet"
+          heading="Store & Transfer"
+          grayedText="Arweave assets"
+          text="With ArConnect's user-friendly interface, you can manage your Arweave balance, view and transfer assets and see your latest transactions. The extension allows you to add all your wallets and switch between them easily"
```
Comment (t8) [inline components/Home/Features/index.tsx]:
```suggestion
          text="With ArConnect's user-friendly interface, you can manage your Arweave balance, view and transfer assets, and see your latest transactions. The extension allows you to add all your wallets and switch between them easily."
```

File: components/Home/Features/index.tsx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+      <section className={styles.features}>
+        <section className={styles.textContainer}>
+          <div className={styles.textWrapper}>
+            <p className={styles.buidl}>Buidl</p>
+            <h3>
+              Develop <span>on ArConnect</span>
+            </h3>
+            <p className={styles.text}>
+              Built by Developers for Developers, ArConnect allows you use
```
Comment (t8) [inline components/Home/Features/index.tsx]:
```suggestion
              Built by devs for devs, ArConnect allows you to use
```

File: components/Home/Features/index.tsx [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+              developer friendly tools to integrate and build on ArConnect
```
Comment (t8) [inline components/Home/Features/index.tsx]:
```suggestion
              developer-friendly tools to integrate and build on ArConnect
```

File: components/Home/Features/index.tsx [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+              across blockchains seamlessly. Explore our Developer documentation
```
Comment (t8) [inline components/Home/Features/index.tsx]:
```suggestion
             seamlessly. Explore our developer documentation
```

File: components/Home/Features/index.tsx [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+              <span className={styles.const}>await </span>
+              arweave.transactions.post(tx);
+            </code>
+          </pre>
+        </section>
+
+        <section className={styles.securityContainer}>
+          <div className={styles.securityInfo}>
+            <p className={styles.safu}>SAFU</p>
+            <h3>Permanent Security</h3>
```
Comment (t8) [inline components/Home/Features/index.tsx]:
```suggestion
            <h3>Top-notch Security</h3>
```

File: components/Home/Features/index.tsx [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+            <p>
+              With ArConnect&apos;s user-friendly interface, you can manage your
```
Comment (t8) [inline components/Home/Features/index.tsx]: This paragraph should be changed to:

```
ArConnect features a robust permissions system, application allowances, and the ability to easily track all ArConnected activity through Dapps.
```

File: components/Home/Features/index.tsx [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+              Arweave balance, view and transfer assets and see your latest
+              transactions. The extension allows you to add all your wallets and
+              switch between them easily. We are also planning to support PST
+              swapping through The Verto Protocol.
+            </p>
+            <Link href="/docs">
```
Comment (t8) [inline components/Home/Features/index.tsx]: This button should be disabled

File: components/Home/Features/index.tsx [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+              <a className={styles.docsBtn}>Learn more</a>
```
Comment (t8) [inline components/Home/Features/index.tsx]:
```suggestion
              <a className={styles.docsBtn}>Learn more (coming soon)</a>
```

File: components/Home/FAQ/index.tsx [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+          <QandA
+            question="How does ArConnect protect my assets?"
+            answer="Coming Soon"
+          />
+        </section>
+      </section>
+
+      <section className={styles.feedbackContainer}>
+        <h2>
+          10,000+ people <span>and counting</span> love ArConnect
```
Comment (t8) [inline components/Home/FAQ/index.tsx]:
```suggestion
          15,000+ people <span>and counting</span> love ArConnect
```

File: components/Home/FAQ/index.tsx [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+            ArConnect is a completely decentralized network built on top of the
+            blockweave. Anyone can help power the exchange. Feel freed to reach
+            out to the community on Discord or Twitter!
+          </p>
+          <Link href="https://verto.exchange/chat">
+            <a className={styles.chatBtn}>Chat with us</a>
+          </Link>
+        </div>
+
+        <section>
```
Comment (t8) [inline components/Home/FAQ/index.tsx:90]: ## Need to figure out before merging

File: components/Home/FAQ/index.tsx [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+};
+
+const FAQ = () => {
+  return (
+    <section className={styles.container}>
+      <section className={styles.faq}>
+        <div>
+          <h2>Here to answer your questions</h2>
+          <p>
+            ArConnect is a completely decentralized network built on top of the
```
Comment (t8) [inline components/Home/FAQ/index.tsx]: This information is wrong. Need to reword. 

```
Feel free to connect with us on Discord, Twitter, or GitHub!
```

File: pages/onboarding/index.tsx [hunk 14]
```diff
@@ -0,0 +1,18 @@
...
+import WelcomePage from 'components/Explore/Welcome';
+
+const Welcome = () => (
+  <>
+    <Head>
+      <title>Welcome to ArConnect</title>
+    </Head>
+    <WelcomePage
+      heading="Welcome!"
+      text="ArConnect was tailor made for you regardless of your background,
```
Comment (t8) [inline pages/onboarding/index.tsx]:
```suggestion
      text="ArConnect was tailor-made for you regardless of your background,
```

File: pages/onboarding/index.tsx [hunk 15]
```diff
@@ -0,0 +1,18 @@
...
+      skill or experience, it’s unbreakable, yet malleable. Please select
```
Comment (t8) [inline pages/onboarding/index.tsx]:
```suggestion
      skills or experience; it’s unbreakable, yet malleable. Please select
```

File: pages/onboarding/index.tsx [hunk 16]
```diff
@@ -0,0 +1,18 @@
...
+      what best describes you"
```
Comment (t8) [inline pages/onboarding/index.tsx]:
```suggestion
      what best describes you."
```

File: pages/onboarding/guide.tsx [hunk 17]
```diff
@@ -0,0 +1,41 @@
...
+      <section className={styles.mainContainer}>
+        <div className={styles.iconContainer}>
+          <img src="/assets/guide.svg" alt="guide dog icon" />
+        </div>
+        <h1>
+          No worries, <span>We&apos;ll guide you</span>
+        </h1>
+        <p>
+          We have curated a set of quick primers to bring you up to speed on
+          crypto, But we understand that you probably need ArConnect for
```
Comment (t8) [inline pages/onboarding/guide.tsx]:
```suggestion
          crypto, but we understand that you may need ArConnect for
```

File: pages/onboarding/guide.tsx [hunk 18]
```diff
@@ -0,0 +1,41 @@
...
+          something immediately
```
Comment (t8) [inline pages/onboarding/guide.tsx]:
```suggestion
          something immediately...
```

File: components/Explore/index.tsx [hunk 19]
```diff
@@ -0,0 +1,94 @@
...
+import useGetBrowser from 'hooks/useGetBrowser';
+
+export const StoreAR = () => {
+  return (
+    <section className={styles.dappsContainer}>
+      <ExplorePage
+        iconUrl="/assets/moneyBag.svg"
+        altText="money bag emoji"
+        heading="Store you first AR tokens"
+        introParagraph="Please click the <span>Download</span> button on this page to get ArConnect for your
```
Comment (t8) [inline components/Explore/index.tsx]:
```suggestion
        introParagraph="Please click the <span>Install ArConnect</span> button on this page to get ArConnect for your
```

File: components/Explore/index.tsx [hunk 20]
```diff
@@ -0,0 +1,94 @@
...
+        browser if you haven’t already, If you have then let’s proceed! You
```
Comment (t8) [inline components/Explore/index.tsx]:
```suggestion
        browser if you haven’t already. If you have then let’s proceed!
```

File: components/Explore/index.tsx [hunk 21]
```diff
@@ -0,0 +1,94 @@
...
+        basically just have to complete the following actions below step by step"
```
Comment (t8) [inline components/Explore/index.tsx]:
```suggestion
        Complete the following actions step-by-step."
```

File: components/Explore/index.tsx [hunk 22]
```diff
@@ -0,0 +1,94 @@
...
+
+export const AccessDapps = () => {
+  return (
+    <section className={styles.dappsContainer}>
+      <ExplorePage
+        iconUrl="/assets/plug.svg"
+        altText="plug emoji"
+        accessDapps
+        heading="Access a Dapp"
+        introParagraph="Please click the <span>Download</span> button on this page to get ArConnect for your browser if you haven’t already, If you have then let’s proceed!
```
Comment (t8) [inline components/Explore/index.tsx]:
```suggestion
        introParagraph="Please click the <span>Install ArConnect</span> button on this page to get ArConnect for your browser if you haven’t already. If you have then let’s proceed!```

**Review by t8 — CHANGES_REQUESTED** [review]
Partial review. Final half coming tomorrow. You can commit most of these suggestions directly :)

**Review by t8 — APPROVED** [review]
LGTM. Amazing work @codingknite! Merge at will.
image

**Pull request merged**